README.md

```diff
@@ -1,39 +1,64 @@
-# Project Template
+# AutoTrace
 
-This is a template repository for CSC 207 projects. 
-This repository contains starter code for a gradle project.
-It also contains workflow documents that give instructions on how to manage your Github repository and how to use Github Projects for efficient collaboration.
+AutoTrace is an Android app designed to provide users with essential information about vehicles by inputting their license plate numbers. It offers a simple and efficient way to retrieve car model and registration date data from relevant databases. Additionally, it provides administrators with the ability to register license plates into the system, along with associated driver and car information.
 
-## Checklist For Your Project
-- [ ] Verify the correct settings for your project repository
-- [ ] Set up Github Projects
-- [ ] Create the implementation plan using issues and Github Projects
-- [ ] Create deveopment branches for your features
-- [ ] Use pull requests to merge finished features into main branch
-- [ ] Conduct code reviews
+## Motivation
 
-**If your team has trouble with any of these steps, please ask on Piazza. For example, with how GitHub Classroom works, your team *may* not have permissions to do some of the first few steps, in which case we'll post alternative instructions as needed.**
+The primary motivation behind AutoTrace is to offer a convenient solution for users who need to identify car models and registration dates based on license plate numbers. This app aims to streamline the process of accessing vehicle information by leveraging databases and providing an intuitive user interface.
```

Comment:
We aimed to create an app that works as a centralized system for displaying issued traffic tickets. The app displays information about each car, such as insurance provider. Another app feature is verifying ownership details by looking up any vehicle registered by the license plate in the database. Users can create an account to register their cars and view filed traffic tickets.